tests/cypress/src/DashTable.ts

```diff
@@ -19,104 +19,200 @@ const getSelector = (state: State) => {
     }
 
-    static selectColumnById(row: number, column: string) {
-        return DashTable.getSelectColumnById(row, column).click();
+    public selectColumnById(row: number, column: string) {
+        return this.getSelectColumnById(row, column).click();
     }
 
-    static getDelete(row: number, editable: State = State.Ready) {
-        return cy.get(`#table ${getSelector(editable)} tbody tr td.dash-delete-cell`).eq(row);
+    public getDelete(row: number, editable: State = State.Ready) {
+        return cy.get(`#${this.id} ${getSelector(editable)} tbody tr td.dash-delete-cell`).eq(row);
     }
 
-    static getSelect(row: number, editable: State = State.Ready) {
-        return cy.get(`#table ${getSelector(editable)} tbody tr td.dash-select-cell`).eq(row);
+    public getSelect(row: number, editable: State = State.Ready) {
+        return cy.get(`#${this.id} ${getSelector(editable)} tbody tr td.dash-select-cell`).eq(row);
     }
 
-    static getActiveCell(editable: State = State.Ready) {
-        return cy.get(`#table ${getSelector(editable)} tbody td.focused`);
+    public getActiveCell(editable: State = State.Ready) {
+        return cy.get(`#${this.id} ${getSelector(editable)} tbody td.focused`);
     }
 
-    static getSelectedCells(editable: State = State.Ready) {
-        return cy.get(`#table ${getSelector(editable)} tbody td.cell--selected`);
+    public getSelectedCells(editable: State = State.Ready) {
+        return cy.get(`#${this.id} ${getSelector(editable)} tbody td.cell--selected`);
     }
 
-    static scrollToTop() {
+    public scrollToTop() {
         cy.get(`.cell.cell-1-1.dash-fixed-content`).invoke(`outerHeight`).then(height => {
             cy.scrollTo(0, -height);
         });
     }
 
-    static scrollToBottom() {
+    public scrollToBottom() {
         cy.get(`.cell.cell-1-1.dash-fixed-content`).invoke(`outerHeight`).then(height => {
             cy.scrollTo(0, height);
         });
     }
 
-    static getCellInLastRowOfColumn(column: number) {
+    public getCellInLastRowOfColumn(column: number) {
         const cellInLastRow = cy.get(`td.dash-cell.column-${column}`).last().then(elem => {
             const lastRow = elem ? elem.attr(`data-dash-row`) : undefined;
             return lastRow ? cy.get(`td.dash-cell.column-${column}[data-dash-row="${lastRow}"`) : undefined;
         });
         return cellInLastRow;
     }
 
-    static getCellFromDataDash(row: number, column: number) {
+    public getCellFromDataDash(row: number, column: number) {
         return cy.get(`td.column-${column}[data-dash-row="${row}"]`);
     }
 
-    static toggleScroll(toggled: boolean) {
+    public toggleScroll(toggled: boolean) {
         cy.get('.row-1').then($el => {
             $el[0].style.overflow = toggled ? '' : 'unset';
         });
     }
 }
+
+export default class DashTable {
+    private static __helper = new DashTableHelper('table');
```

Comment:
would it not work to just say `export default const DashTable = new DashTableHelper('table');`?

Reply:
😞 very much so